python/triton/language/core.py

```diff
@@ -9,6 +9,8 @@
 
 T = TypeVar('T')
 
+TRITON_MAX_TENSOR_NUMEL = 131072
```

Comment:
I'm new to triton, so apologies for basic question. 

Should this number be device and dtype specific? I am guessing this has something to do with L1 cache size?